content/en/metrics/overview.md

```diff
@@ -0,0 +1,138 @@
+   - [Custom checks][8], which are used to collect metrics from custom applications or unique systems. You define the logic for the check yourself in the Agent's configuration files. See [Writing a Custom Agent Check][9] for more information.
+   - [Marketplace integrations][10] installed on the Agent. The [Datadog Marketplace][11] is a digital marketplace where Technology Partners can list their paid offerings to Datadog users.
+{{% /collapse-content %}}
+
+{{% collapse-content title="Cloud integrations" level="h4" %}}
+Also known as authentication-based integrations, these integrations are set up in Datadog. You provide the credentials for obtaining metrics, and Datadog makes API calls on your behalf to collect them. Common examples are cloud provider integrations, Slack, and PagerDuty. See [API-based integrations][12] in the Developers documentation for more information.
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Datadog API" level="h4" %}}
+You can send metrics directly to the [Metrics API][13].
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Metrics from other Datadog products" level="h4" %}}
+Some Datadog products, such as Synthetic Testing, provide standard metrics by default. Other products can be configured to generate custom metrics. This section covers the available options and provides links to relevant documentation.
+
+##### Logs
+
+[Generate custom metrics from ingested logs][14] to summarize data from all the logs ingested by Datadog. This allows you to visualize and alert on log data that's important to your environment, even if the logs are not indexed for long-term search.
+
+##### APM
+
+[Generate custom metrics from your ingested spans][15] to visualize anomalies and trends across any parameters that are important to your business context.
+
+##### Real User Monitoring (RUM)
+
+[Generate custom metrics from RUM events][16] to summarize the data from your RUM events, so you can visualize and alert on the user behaviors that are most impactful to your organization.
+
+##### Processes
+
+[Generate custom process-based metrics][17] to monitor the resource consumption of your processes, as well as any other process-related behavior that might be important to your business needs.
+
+##### Events
+
+[Generate custom event-based metrics][18] for visibility into monitor alerts or any other event-based data ingested by Datadog.
+
+{{% /collapse-content %}} 
+
+### Configurable Processing
+
+The **Configurable Processing** column shows an overview of the configuration options available for your metrics. Click on any of the options for more information and a link to the relevant configuration screen.
+
+{{% collapse-content title="Optimize your custom metrics costs with Metrics without Limits™" level="h4" %}}
+[Metrics without Limits™][19] helps you manage custom metric costs by indexing only the metric tags most valuable to your organization.
```

Comment:
```suggestion
[Metrics without Limits™][19] helps you manage custom metric costs by indexing only the metric tags most valuable to your organization. Your use of Metrics without Limits is reflected in the top section of the Overview page as "Selected Metrics". For more best practices on how to manage your custom metrics costs, please refer to our end to end guide here[]
```

Link to governance guide : https://docs.datadoghq.com/metrics/guide/custom_metrics_governance

Reply:
Link added in [803ca9f](https://github.com/DataDog/documentation/pull/25830/commits/803ca9ff51587e3b1aad595e46e6e064d172caed)